### ISSUE TITLE
can`t user es7 destructuring assignments 

### ISSUE THREAD

**Issue by nokisnojok** [parcel-bundler/parcel#1221]
**Choose one:** is this a 🐛 bug report or 🙋 feature request?

### 🎛 Configuration (.babelrc, package.json, cli command)

mian.js
```js
var { a, b, ...other } = { a: 1, b: 2, c: 3, d: 4 }
```

.babelrc
```js
{
    "presets": [
        "es2015",
        "react"
    ]
}
```
### 😯 Current Behavior
parcel ./main.js

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.7.1
| Node             | 8.11.1
| npm/Yarn         | npm 5.6.0
| Operating System | windows10

**Comment by bary12** [comment]
Object rest spread is an experimental feature available through [Babel Preset Stage 3](https://babeljs.io/docs/plugins/preset-stage-3/) or [as a standalone plugin](https://babeljs.io/docs/plugins/transform-object-rest-spread/) and is not shipped with babel-preset-es2015. add wither the stage 3 preset or the plugin to your .babelrc file.

**Comment by edmorley** [comment]
Object rest spread is now stage 4:
https://github.com/tc39/proposal-object-rest-spread#status-of-this-proposal

...and included in Babel 7's `@babel/preset-env` (as of `v7.0.0-beta.3`):
https://github.com/babel/babel/blob/21b03c35d38f1d18cb535434e76defbc803b77a3/packages/babel-preset-env/data/plugins.json#L227-L235

As such this will be fixed by #868.
